content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,117 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you’ve just created the reference entity `Brand`, you’ve defined it’s records' attributes and now it’s time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
+
+To add a new brand to the reference entity `Brand`:
+1. Go to your `Brand` entity (you are automatically on the records grid)
+1. Then click on the `Add new` button, choose a label for the record (the code is automatically generated but you can change it if needed).
+
+::: tips
+If you have several records to add:
+1. Tick `Create another record`
+1. Input the first record's label and press "Enter"
+1. Input the second record's label and press "Enter"
+1. Do it for each record then save.  
```

Comment:
By input do you mean "add"? If that is the case it could be better to say : 
1. Tick `Create another record`
1. Add the label for each record and press "Enter"
1. Save the records

Reply:
I changed a bit the steps